python/plugins/processing/algs/gdal/contour.py

```diff
@@ -162,6 +167,9 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
 
         if offset:
             arguments.append('-off {}'.format(offset))
+        
```

Comment:
Before this, you'll need to retrieve the options value. See https://github.com/qgis/QGIS/blob/master/python/plugins/processing/algs/gdal/ClipVectorByMask.py#L86

    options = self.parameterAsString(parameters, self.OPTIONS, context)
    if options:
      ...

Reply:
That's great Nyall - I've included your suggestions in the version in my fork (and they work fine in a local copy)
I've also added a unit test to GdalAlgorithmsTest.py in my fork (lines 592-603)
What's the next step - do I have to resubmit the pull request?
Thanks for your patience!
Geoff

Reply:
Nothing more for you to do, until the Travis CI results come in. If they are all green I'll merge, otherwise I'll advise what the next step is.